SPECS/python-tensorboard/python-tensorboard.spec

```diff
@@ -6,24 +6,32 @@ TensorBoard is a suite of web applications for inspecting and understanding your
 
 Summary:        TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs
 Name:           python-%{pypi_name}
-Version:        2.11.0
-Release:        4%{?dist}
+Version:        2.16.2
+Release:        1%{?dist}
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://github.com/tensorflow/tensorboard
-Source0:        https://github.com/tensorflow/tensorboard/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        %{name}-%{version}-cache.tar.gz
+# This source also contains the dependencies required for building tensorboard
+Source0:        %{_distro_sources_url}/%{name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+Patch0:         0000-Use-system-package.patch
 BuildRequires:  python3-setuptools
```

Comment:
could you alphabetize the list of BuildRequires? should make it easier to read

Reply:
alphabetized BRs